### PR TITLE
cleanup more components to use radium

### PR DESCRIPTION
**Pull request by Bjvanminnen**
Make CollapserIcon a stateless functional component

Use Radium in a bunch of places

Make HeightResizer's position prop be the position of the bottom of the resizer instead of the top.

### CODE REVIEW

File: apps/src/templates/instructions/HeightResizer.jsx [hunk 1]
```diff
@@ -86,9 +87,9 @@ var HeightResizer = React.createClass({
   },
 
   render: function () {
-    var mainStyle = _.assign({}, styles.main, {
-      top: this.props.position
-    });
+    var mainStyle = [styles.main, {
+      top: this.props.position - RESIZER_HEIGHT
+    }];
```
Comment (Bjvanminnen) [inline apps/src/templates/instructions/HeightResizer.jsx:92]: For cases where we're combining a bunch of static styles (like in MarkdownInstructions) the array of objects pattern supported by Radium seems clearly better than doing _.assign. In the case where we have dynamic styles, and thus end up creating objects on the fly, it's less clearly better, but I think I still prefer this approach as it's less verbose. Thoughts?

Comment (islemaster) [inline apps/src/templates/instructions/HeightResizer.jsx:92]: Agreed, this is less verbose.  Go for it.

File: apps/src/templates/instructions/MarkdownInstructions.jsx [hunk 2]
```diff
@@ -18,17 +20,14 @@ var styles = {
...
-
   return (
     <div
       className='instructions-markdown'
-      style={style}
+      style={[
+        styles.standard,
+        props.inTopPane && styles.inTopPane,
+        props.markdownClassicMargins && styles.classic
+      ]}
```
Comment (islemaster) [inline apps/src/templates/instructions/MarkdownInstructions.jsx:30]: It's pretty awesome to be able to express this way.

**Comment by islemaster** [comment]
Check CircleCI

```
>> Error: Cannot find module 'Radium' from '/home/ubuntu/code-dot-org/apps/build/js/templates/instructions'
```

Otherwise, LGTM

**Pull request merged**